src/OpenTK.NT/Native/User32/Structs/RawHid.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+using DWORD = System.UInt32;
+
+namespace OpenTK.NT.Native
+{
+    /// <summary>
+    /// The RawHID structure describes the format of the raw input
+    /// from a Human Interface Device (HID).
+    /// </summary>
+    /// <remarks>
+    /// Each WM_INPUT can indicate several inputs, but all of the inputs
+    /// come from the same HID. The size of the bRawData array is
+    /// dwSizeHid * dwCount.
+    /// </remarks>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct RawHid
+    {
+        /// <summary>
+        /// Size, in bytes, of each HID input in bRawData.
+        /// </summary>
+        public DWORD SizeHid;
+
+        /// <summary>
+        /// Number of HID inputs in bRawData.
+        /// </summary>
+        public DWORD Count;
+
+        /// <summary>
+        /// Raw input data as an array of bytes.
+        /// </summary>
+        public byte RawData;
```

Comment:
Is this a byte-sized field, or a pointer to a byte?

Reply:
I honestly don't quite know.

The [doc](https://msdn.microsoft.com/en-us/library/windows/desktop/ms645549(v=vs.85).aspx) specifies it as a byte array with a length of 1, and then further down says that the length of `RawData` is dependent on the other 2 fields.

Reply:
Judging by some other sources (https://stackoverflow.com/questions/48776221/rawinput-rawhid-structure-in-c-sharp-byte-array and https://www.codeproject.com/Articles/17123/Using-Raw-Input-from-C-to-handle-multiple-keyboard), this is correct, but that seems very strange. Probably merits testing.

Reply:
Yes, needs testing.
From the docs it seems like the pointer assumption is correct too (and it's a length 1 array so the sizeof() calculation is easier or something)